### DISCUSSION TITLE
Be able to upload .iso in an automated way

### DISCUSSION THREAD

**Discussion by LostRager** [Audionut/Upload-Assistant#46] [Ideas]
Having the option to be able to upload .iso's automated would be nice. 

I think it should be pretty straightforward in theory.

For example on windows mounting the disk as Z: and then running the normal routine from Z: instead of the iso path like mediainfo/bdinfo, screenshots, etc..
Then dismount it and use the iso path for hashing. Repeat if there are more then 1 disk, like it would do with dvd/bluray.

Windows (works from both cmd and ps):
`PowerShell Mount-DiskImage -ImagePath "test.iso"`
`PowerShell Dismount-DiskImage -ImagePath "test.iso"`

Linux:
`sudo mkdir /media/iso`
`sudo mount /path/to/image.iso /media/iso -o loop`
`sudo umount /media/iso`

**Comment by Audionut** [comment]
I'm not overly fond of the code complexity that will be required to support this. It would require the additional checks to prohibit users from pushing ISO where prohibited, otherwise it'll likely end up annoying a bunch of site staffers.

I know 3D is usually required in ISO format, but is there really a huge market for ISO otherwise?

**Reply by LostRager** [reply]
Correct, the only iso's that should be uploaded would be 3D disks. 

I myself despise .iso disks, but it has to be uploaded like that if its 3D :(
